frontend/fingertips-frontend/components/linechartsOptions/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { WeatherForecast } from '@/generated-sources/api-client';
+import Highcharts from 'highcharts';
+import { HighchartsReact } from 'highcharts-react-official';
+
+interface LineChartProps {
+  data: WeatherForecast[];
+  title?: string;
+  xAxisTitle?: string;
+  yAxisTitle?: string;
+}
+
+export function LineChart({
+  data,
+  title = 'Weather Forecast',
+  xAxisTitle = 'Date',
+  yAxisTitle = 'Temperature (°C)',
```

Comment:
I don't think we need default values here. I don't think there are any values that can sensibly be applied to all uses of this component, so we should require them to always be provided.